src/app/docs/box/box-docs.component.html

```diff
@@ -29,14 +29,162 @@
+    </sky-docs-design-guideline>
+
+    <sky-docs-design-guideline
+      heading="Don't use when"
+      headingStyle="danger"
+    >
+      <p>
+        Don't use boxes to contain individual content elements. Use repeaters or other layout approaches instead.
+      </p>
+      <sky-docs-design-guideline-thumbnail
+        caption="Don't use boxes to contain individual content elements."
+        captionType="danger"
+        imageSource="~/assets/img/guidelines/box/box-dont-individual-element.png"
+      >
+      </sky-docs-design-guideline-thumbnail>
+    </sky-docs-design-guideline>
+
+    <sky-docs-design-guideline>
+      <p>
+        Don't use boxes to contain sections in modal forms or split view workspaces. Use appropriate headers and spacing to separate the content instead.
+      </p>
+      <sky-docs-design-guideline-thumbnail
+        caption="Don't use boxes to contain sections in forms."
+        captionType="danger"
+        imageSource="~/assets/img/guidelines/box/box-dont-form.png"
+      >
+      </sky-docs-design-guideline-thumbnail>
+      <sky-docs-design-guideline-thumbnail
+        caption="Don't use boxes to contain sections in split views."
+        captionType="danger"
+        imageSource="~/assets/img/guidelines/box/box-dont-split-view.png"
+        showBorder="true"
+      >
+      </sky-docs-design-guideline-thumbnail>
+    </sky-docs-design-guideline>
+
+    <sky-docs-design-guideline>
+      <p>
+        Don't add boxes to pre-defined page template sections that don't have boxes built in.
+      </p>
+      <sky-docs-design-guideline-thumbnail
+        caption="Don't add boxes to the sections in the right panel of an action hub."
+        captionType="danger"
+        imageSource="~/assets/img/guidelines/box/box-dont-related-recent.png"
+        showBorder="true"
+      >
+      </sky-docs-design-guideline-thumbnail>
+      <sky-docs-design-guideline-thumbnail
+        caption="Don't add a box to contain the list on a list page."
+        captionType="danger"
+        imageSource="~/assets/img/guidelines/box/box-dont-list.png"
+        showBorder="true"
+      >
+      </sky-docs-design-guideline-thumbnail>
+    </sky-docs-design-guideline>
+
+  </sky-docs-demo-page-section>
+
+  <sky-docs-demo-page-section
+    heading="Anatomy"
+  >
+    <sky-docs-anatomy
+      imagePath="~/assets/img/guidelines/box/box-anatomy.png"
+    >
+      <sky-docs-anatomy-item>Box container</sky-docs-anatomy-item>
+      <sky-docs-anatomy-item isOptional="true">Elevation shadow</sky-docs-anatomy-item>
+    </sky-docs-anatomy>
+  </sky-docs-demo-page-section>
+   
+  <sky-docs-demo-page-section
+    heading="Layout"
+  >
+
+    <sky-docs-design-guideline>
+      <p>
+        Add <code>class="sky-padding-even-xl"</code> to the box to ensure proper spacing of the content from the borders of the box.
+      </p>
+      <sky-docs-design-guideline-thumbnail
+        caption="Add XL padding to the box."
```

Comment:
My thought around referencing XL is that we want the consumer to use the appropriate classes/variables so if we make updates they propagate through. What do you think?

Reply:
I hadn't thought about propagating changes, and my edits are definitely a bad call for that. I didn't like "XL padding" and "XL spacing" since we don't use that terminology in the spacing guidelines at https://developer.blackbaud.com/skyux-modern/design/spacing, but if you think we're good, I'm sure people can figure it out.

```suggestion
        caption="Use XL padding within boxes."
```